cmd/lakectl/cmd/repo_create.go

```diff
@@ -29,12 +29,17 @@ var repoCreateCmd = &cobra.Command{
 		if err != nil {
 			DieErr(err)
 		}
+		readonlyRepository, err := cmd.Flags().GetBool("read-only")
```

Comment:
why we enable creating a read-only repository? we plan to enable half updates?
if we can update the read-only property on a repository we don't need to create one when it is `on` unless we restore or edit.